python/tests/test_cdh_utils.py

```diff
@@ -58,7 +58,7 @@ def test_find_default_predictors():
 
 
 def test_file_not_found():
-    with pytest.raises(FileNotFoundError):
+    with pytest.raises(Exception):
```

Comment:
Again, I don't like general Exceptions. Let's try to specify those. FileNotFoundError is much nicer, imo.